src/mapping/nav2_map_server/src/map_reps/occ_grid_server.cpp

```diff
@@ -254,14 +254,26 @@ void OccGridServer::LoadMapFromFile(const std::string & map_name_)
   }
 
   SDL_FreeSurface(img);
+
+  rclcpp::Clock clock;
+  map_msg_.info.map_load_time = clock.now();
+  map_msg_.header.frame_id = frame_id;
+  map_msg_.header.stamp = clock.now();
```

Comment:
Should we make both of these the same time? It seems a bit odd to have successive clock.now() calls with slightly different times.

Reply:
I'm not sure. This is what is done in the ROS1 map_server. I suppose we should start by asking what information map_load_time and stamp are trying to convey. The stamp I would think is the time the message is generated, and perhaps the map_load_time is however long it took to create the map from file (arguably could they be roughly the same time?).